src/components/searchfield/searchfield.js

```diff
@@ -36,9 +37,20 @@ export default class SearchField {
     this.matchAllQuery = matchAllQueryWhenSearchFieldEmpty;
     this.minLengthToShowResults = conf.minLengthToShowResults || 1;
     this.firstRenderDone = false;
-    this.firstSelectorBindDone = false;
     this.onSearch = onSearch;
 
+    const minLengthToShowResults = conf.minLengthToShowResults || 0;
+    this.reduxStore.dispatch(
+      autocompleteMinLengthRequired({
+        minLengthRequired: minLengthToShowResults
+      })
+    );
+    this.reduxStore.dispatch(
```

Comment:
I think you are fine with just 1 action and one location for storing that boolean value in the redux state.  (You are currently only reading from the autocomplete.minLengthToShowResults state, i.e. the search.minLengthToShowResults is set, but not used) 

I would argue it should be that probably it would be more logical to use the search reducer for it, because the option is set on the SearchField component by the user. 

But perhaps it would also make sense to introduce a new reducer for general settings. But maybe that is a bit too big for this pr.

Reply:
I'll remove both and place it in `keyword` state maybe?
Since this setting is set by `searchField`

Reply:
Sounds good.